File: class_7/class_7/app/src/main/java/com/alejouribesanchez/class_7/Main7Activity.java
package com.alejouribesanchez.class_7;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class Main7Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main7);

        View layout = (View)findViewById(R.id.activity_main7);
        layout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Main7Activity.this, Main8Activity.class));
            }
        });
    }
}

File: class_7/class_7/app/src/main/java/com/alejouribesanchez/class_7/Main7Activity.kt
package com.alejouribesanchez.class_7

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class Main7Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main7)

        findViewById(R.id.activity_main7).setOnClickListener { startActivity(Intent(this@Main7Activity, Main8Activity::class.java)) }
    }
}
